dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-datax.ts

```diff
@@ -131,7 +131,8 @@ export function useDataX(model: { [field: string]: any }): IJsonItem[] {
     'CLICKHOUSE',
     'DATABEND',
     'HIVE',
-    'PRESTO'
+    'PRESTO',
+    'XUGU'
```

Comment:
hi @mrliufox  if expose ```XUGU``` in the font end ,and use rdbmsreader or rdbmswriter in the backend ,please post a screenshot of the results of running xugu read and write locally using DataX's rdbms type.

Reply:
i have removed xugu from datax in the font end